packages/users/src/users.ts

```diff
@@ -208,6 +210,22 @@ export class Users extends GrpcAuthentication {
     return super.getTokenChallenge(publicKey, callback)
   }
 
+  /**
+   * GetUsage returns current billing and usage information.
+   *
+   * @example
+   * ```typescript
+   * import { Users } from "@textile/hub"
+   *
+   * async function example(users: Users) {
+   *    const usage = await users.getUsage()
+   * }
+   * ```
+   */
+  async getUsage(): Promise<GetUsageResponse> {
```

Comment:
Side note: I wonder if we can show example output in these examples?

Reply:
yeah, will add that as a follow-up.